cloud/deployment/deployment_variable.go

```diff
@@ -91,7 +91,7 @@ func VariableModify(
 	environmentVariablesObjects := []astroplatformcore.DeploymentEnvironmentVariable{}
 
 	// get deployment
-	currentDeployment, err := GetDeployment(ws, deploymentID, deploymentName, false, nil, platformCoreClient, nil)
+	currentDeployment, err := GetDeployment(ws, deploymentID, deploymentName, false, nil, platformCoreClient, coreClient)
```

Comment:
is there a possible unit test, we can add for this case?

Reply:
Adding it right now...